tests/stdlib/topenssl.nim

```diff
@@ -0,0 +1,19 @@
+discard """
+  matrix: "-d:ssl"
+"""
+import openssl
+
+# bug #16308
+block:
+  var rsa: PRSA
+  var file: File
+

```

Comment:
add a note explaining how to (re-) generate topenssl_rsa_private_key.pem and topenssl_rsa_public_key.pem